From 89d5ccaf224c422d18296126b1978a7646951889 Mon Sep 17 00:00:00 2001
From: Marek 'marx' Grac <mgrac@redhat.com>
Date: Wed, 16 Dec 2015 10:21:47 +0100
Subject: [PATCH] fence_ipmilan: 'diag' action should not require setting
 --method=cycle

When action is 'diag', set --method to cycle automatically.
---
 fence/agents/ipmilan/fence_ipmilan.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/fence/agents/ipmilan/fence_ipmilan.py b/fence/agents/ipmilan/fence_ipmilan.py
index 2901492..f132ab7 100644
--- a/fence/agents/ipmilan/fence_ipmilan.py
+++ b/fence/agents/ipmilan/fence_ipmilan.py
@@ -189,6 +189,7 @@ This agent calls support software ipmitool (http://ipmitool.sf.net/)."
 		# Diag is a special action that can't be verified so we will reuse reboot functionality
 		# to minimize impact on generic library
 		options["--action"] = "reboot"
+		options["--method"] = "cycle"
 		reboot_fn = reboot_diag
 
 	result = fence_action(None, options, set_power_status, get_power_status, None, reboot_fn)
-- 
2.4.3

